fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/DefaultAccountProvider.java

```diff
@@ -34,9 +34,12 @@
 @Component
 public class DefaultAccountProvider extends BaseProvider implements AccountProvider {
 
+  private final ClouddriverService clouddriverService;
+
   @Autowired
-  @Setter
-  private ClouddriverService clouddriverService;
+  public DefaultAccountProvider(ClouddriverService clouddriverService) {
```

Comment:
I'd suggest using Lombok's `@RequiredArgConstructor` to make this easier.

https://projectlombok.org/features/Constructor.html